docs/modules/processor/nav.adoc

```diff
@@ -4,3 +4,4 @@
 ** xref:extract-api.adoc[Extract information from a document]
 ** xref:manipulate-api.adoc[Manipulate a document]
 ** xref:logging-api.adoc[Logging]
+* https://asciidoctor.github.io/asciidoctor.js/2.2.3/[JSDoc]
```

Comment:
You might want to think about making the version an attribute.

Reply:
I wasn't sure whether we could use AsciiDoc attributes in `nav.adoc`

Reply:
Yep, they work there.